README.md

```diff
@@ -29,25 +19,45 @@ Once you've written your provider, you'll want to [publish it on the Terraform R
 go install
 ```
 
-## Adding Dependencies
+## Configuring the provider
```

Comment:
Is the model for consuming this provider forking the repo and rebasing over time? If so, a little bit less polished than a basic scaffold that consumes a some common go library that can be updated that way, but perhaps that's a nice-to-have more than a blocker for someone to consume and use the provider.

Reply:
That's the current model. 

Ideally, we should move towards a common library. I'm not entirely sure at a tactical level what that would look like though and how you would publish the library.